meta-luneos/recipes-android/android-tools/android-tools-conf/0001_Enable_FunctionFS.install_patch

```diff
@@ -0,0 +1,18 @@
+--- a/android-gadget-setup
++++ b/android-gadget-setup
+@@ -6,6 +6,15 @@
+ # model=$(getprop ro.product.model Android)
+ # serial=$(getprop ro.serialno 0123456789ABCDEF)
+ 
++#below are now needed in order to use FunctionFS for ADB, tested to work with 3.4+ kernels
++if [ grep -q functionfs /proc/filesystems ]; then
++    mkdir -p /dev/usb-ffs/adb 
++    mount -t functionfs adb /dev/usb-ffs/adb
++    #android-gadget-setup doesn't provide below 2 and without them it won't work, so we provide them here.
++    echo adb > /sys/class/android_usb/android0/f_ffs/aliases
++    echo ffs > /sys/class/android_usb/android0/functions 
```

Comment:
This should probably be "echo mtp,ffs" instead, if you want mtp too. To be tested.